cli/hclfmt.go

```diff
@@ -62,6 +66,16 @@ func formatTgHCL(terragruntOptions *options.TerragruntOptions, tgHclFile string)
 	}
 
 	newContents := hclwrite.Format(contents)
+	checkMode := util.ListContainsElement(terragruntOptions.TerraformCliArgs, "-check")
```

Comment:
I'm not sure I like this. On the one hand, it's a nice way to keep the code simple. On the other hand, this is highly dependent on how we are parsing the args and segregating what counts as Terragrunt args and what counts as Terraform args. Not to mention the awkwardness where this is a terragrunt specific command inspecting args for terraform commands...

Could use a second opinion here. What do you think @brikis98 ?

Also, what do you think about using a different command name for this mode? E.g something like `check-hclfmt`.

Reply:
Yeah, totally. I already asked about in the issue. :-) 
Personally, I don't like the mixed context here, e.g. lookup Terraform structure to validate Terragrunt specific option.

Reply:
Other way, an additional command `check-hclfmt` sounds odd. I don't mind to follow already existing practice where Terragrunt specific options use prefix `--terragrunt`.

Reply:
I think the "idiomatic" way to handle this in the Terragrunt code base is to add this flag to `TerragruntOptions` and update the parse method for it to read out this arg. We do that for other Terragrunt-specific args already.

Reply:
@brikis98 Yay! Thank you!